src/pages/library/DataBrowserPreview.js

```diff
@@ -73,6 +73,18 @@ const DatasetPreviewSelector = ({ access, dataset, selectedTable, setSelectedTab
   })],
   [Utils.DEFAULT, undefined])
 
+export const formatTableCell = ({ cellKey, cellContent, rowIndex, table, setViewJSON }) => {
+  const parsableCellContent = _.isObject(cellContent) ? JSON.stringify(cellContent) : cellContent
+  const maybeJSON = Utils.maybeParseJSON(parsableCellContent)
+  return Utils.cond(
+    [!Utils.cantBeNumber(cellContent), () => cellContent],
```

Comment:
I didn't notice this before but.. `cantBeNumber()`?? One would hope for `!canBeNumber()` instead.

Reply:
Yeah, sadly :| It looks like it is used elsewhere so I would hesitate to change it.